mappings/net/minecraft/world/chunk/WorldChunk.mapping

```diff
@@ -78,11 +82,13 @@ CLASS net/minecraft/class_2818 net/minecraft/world/chunk/WorldChunk
 		ARG 1 world
 	METHOD method_20530 disableTickSchedulers ()V
 	METHOD method_31712 removeAllBlockEntities ()V
-	METHOD method_31713 updateAllBlockEntityTickers ()V
+	METHOD method_31713 updateAllBlockEntities ()V
```

Comment:
Nice rename!

Reply:
Yes, it updates the BE Game Event Listeners too now